src/main/scala/latis/service/hapi/Parameter.scala

```diff
@@ -39,8 +39,9 @@ object Parameter {
     deriveEncoder[Parameter].mapJsonObject { obj =>
       JsonObject.fromIterable {
         obj.toList.filter {
-          case ("fill", _) => true
-          case (_, v)      => !v.isNull
+          case ("units", _) => true
```

Comment:
Needed to explicitly include `null` entry for units.

Reply:
Could you update the Scaladoc above to mention allowing units to be null?